tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -246,46 +323,22 @@ def test_fio_workload_simple(self, ripsaw, es, interface, io_pattern):
                 log.info("Bench pod not ready yet")
 
         # Getting the start time of the test
-        start_time = time.strftime("%Y-%m-%dT%H:%M:%SGMT", time.gmtime())
+        self.start_time = time.strftime("%Y-%m-%dT%H:%M:%SGMT", time.gmtime())
```

Comment:
You can do `self.start_time = datetime.now` instead

Reply:
the output is different and it will need to be formatted anyway, so i don't see any reason to use different function

Reply:
Ok